examples/all.yaml

```diff
@@ -6,6 +6,12 @@ deploy_cli_plugins: []  # List of cli-plug-ins to be deploy and test e.g ["custo
 user_custom_plugin: ""  # 'oc-' prefixed user defined custom plug-in name e.g. oc-my-plugin.sh
 golang_tar_url: ""  # Golang tar URL to install golang version >= 1.18
 
+# User defined project monitoring vars
+enable_user_defined_project_monitoring: false  # Set true to enable role execution
+user_defined_app_name: ""  # User defined app name e.g. prometheus-example-app
+user_defined_namespace: ""  # User defined namespace e.g. test-namespace
+service_monitor_name: ""  # User defined name for ServiceMonitor object e.g. prometheus-example-app-monitor
+
```

Comment:
Instead of putting the eg value comments, please put it within quotes. This is an example file anyway.

Reply:
The value has been put in the quotes instead of mentioning in the comment as an example.